apps/widget-configurator/src/app/configurator/hooks/useWidgetParamsAndSettings.ts

```diff
@@ -59,6 +61,14 @@ export function useWidgetParams(configuratorState: ConfiguratorState, isDappMode
       },
 
       hideConnectButton: isDappMode,
+
+      partnerFee:
+        partnerFeeBps && partnerFeeBps > 0
```

Comment:
Is the first part needed?

`undefined > 0 === false`

Reply:
Fixed